### DISCUSSION TITLE
Did anyone face any issue with driver after upgrading to 11.6.5 from 11.6.4? Is driver working in 11.6.5?

### DISCUSSION THREAD

**Discussion by NavinSrivastava** [chris1111/Wireless-USB-Big-Sur-Adapter#197] [Q&A] [answered]
I have driver working in 11.6.4 Thanks a ton to Chris to get this driver. 
I want to upgrade to 11.6.5 from my current version 11.6.4. 
Did anyone face any issue with driver after upgrading to 11.6.5 from 11.6.4? Is driver working in 11.6.5?

**Comment by chris1111** [comment, marked as answer]
If you have problem,  Uninstall then reinstall utility 
This drivers work from Catalina to Monterey 12

**Comment by Gnarlodious** [comment]
Works real good for me on my 10 year old Macbook Pro running 11.6.8 Big Sur. Not working through my powered USB hub, it is recognized as connected but I get dialog to enable it in Network devices, where it does not appear.
